src/supplementField.ts

```diff
@@ -198,12 +198,14 @@ export function supplementFieldOption(fieldOption: FieldOption, fieldDef: FieldD
   if (!config.countTitle) {
     config.countTitle = vl.config.defaultConfig.countTitle; // use vl default countTitle
   }
-  supplementedFieldOption.title = fieldOption.title ?
-    fieldOption.title : vl.fieldDef.title(fieldDef, config);
+  let defaultTitle: string = vl.fieldDef.title(fieldDef, config);
+  if ((fieldDef as any).axis && (fieldDef as any).axis.title) {
```

Comment:
Why is the `as any` necessary?

Reply:
How can you use it if it's not a valid property? I think any is an indicator of an error in the logic.

Reply:
You can use a type guard for position fielddef (or add one if we dont have one yet)